.gitignore

```diff
@@ -2,5 +2,4 @@
 .Rhistory
 .RData
 .Ruserdata
-/rsconnect/
-clef.txt
```

Comment:
Do you mind if we keep this entry?
It is useful to prevent people from committing secrets by mistake.